react-app/src/components/ProposalScreen/ProposalCard.tsx

```diff
@@ -14,7 +14,7 @@ import LocalizedText from "../common/Localized/LocalizedText";
 import ColorBar, { ColorBarData } from "../common/ColorBar/ColorBar";
 import { ProposalInsight } from "./ProposalInsight";
 
-function getProposalStatusBadgeConfig(
+export function getProposalStatusBadgeConfig(
   status: ProposalStatus
 ): [MessageID, BadgeColor] {
   switch (status) {
```

Comment:
If the function becomes reused from a specific screen we probably should extract it into somewhere commonly accessible